app/controllers/applications_controller.rb

```diff
@@ -46,6 +44,14 @@ def destroy
 
   private
   def application_params
-    params.permit(:name, :street_address, :city, :state, :zipcode, :description, :good_owner_comments, status: "In Progress")
+    params.permit(
+      :name,
+      :street_address,
+      :city,
+      :state,
+      :zipcode,
+      :description,
+      :good_owner_comments,
+      status: "In Progress") # is it best practice to hard code this or is there a better way to do this?
```

Comment:
That is true, but I think having a method to update the status to In Progress would be slightly more flexible if we wanted to use this application_params for something other than create. It works either way but I feel like its best practice to not hard code this stuff in wherever possible, because if we wanted to change the wording of "In Progress" at some point, we'd have to change it in a handful of places, rather than just one method and a few tests. I'm being nit-picky here but trying to think about developer empathy and maintainability long term! I think we can leave it though since its just a small project :)